pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
```

Comment:
could you add this? :D https://github.com/uc-cdis/gen3utils/pull/33#discussion_r452527698